### ISSUE TITLE
What are the main differences in train #5 and #6 releases?

### ISSUE THREAD

**Issue by gertelrina** [ultralytics/yolov5#9657]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

What do I need change in train pipeline to getting the same metrics as in #6 rel(without pretrain)?

**Comment by glenn-jocher** [comment]
@gertelrina see README training section for commands to train YOLOv5.

**Comment by gertelrina** [comment]
Yea, I have seen and used this, but I don’t understand how did you manage to raise the mAP, bcs compare two version releases is really long task, so I hoped that u can answer, thank you😊

**Comment by glenn-jocher** [comment]
@gertelrina thanks for your interest! You can improve the mAP by optimizing the training parameters such as learning rate, data augmentation, model size, and the number of training epochs. Additionally, fine-tuning the pre-trained model on a similar dataset can also help improve performance. The changes between releases #5 and #6 are documented in the release notes and the model configurations in the repository. I'd like to highlight that the advancements in performance are primarily due to the collective efforts and contributions of the YOLO community and the diligent work by the Ultralytics team. Good luck with your training!
